cmd/neofs-adm/internal/modules/morph/config.go

```diff
@@ -69,9 +70,7 @@ func dumpNetworkConfig(cmd *cobra.Command, _ []string) error {
 			netmapEigenTrustIterationsKey,
 			netmapEpochKey, netmapInnerRingCandidateFeeKey,
 			netmapMaxObjectSizeKey, netmapWithdrawFeeKey:
-			nbuf := make([]byte, 8)
-			copy(nbuf, v)
-			n := binary.LittleEndian.Uint64(nbuf)
+			n := binary.LittleEndian.Uint64(slices.Clone(v[:8]))
```

Comment:
But why is it copied at all?

Reply:
Likely because it can be _less_ than 8 bytes in which case current code will panic.

Reply:
ok, reverted this change as the minimal possible solution here (it would be strange if we have less than 8 bytes here though)